sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -106,24 +107,24 @@ az cognitiveservices account create \
 For more information about creating the resource or how to get the location and sku information see [here][azure_cli]
 
 ### Authenticate the client
-In order to interact with the Text Analytics service, you will need to create an instance of the Text Analytics client,
+In order to interact with the Language service, you will need to create an instance of the Language service client,
 both the asynchronous and synchronous clients can be created by using `TextAnalyticsClientBuilder` invoking `buildClient()`
 creates a synchronous client while `buildAsyncClient()` creates its asynchronous counterpart.
 
 You will need an **endpoint** and either a **key** or **AAD TokenCredential** to instantiate a client object. 
 
 #### Looking up the endpoint
-You can find the **endpoint** for your Text Analytics resource in the [Azure Portal][azure_portal] under the "Keys and Endpoint",
-or [Azure CLI][azure_cli_endpoint].
+You can find the **endpoint** for your Language service resource in the [Azure Portal][azure_portal] under the
+"Keys and Endpoint", or [Azure CLI][azure_cli_endpoint].
 ```bash
-# Get the endpoint for the text analytics resource
+# Get the endpoint for the Language service resource
 az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "endpoint"
 ```
 
-#### Create a Text Analytics client with key credential
+#### Create a Language service client with key credential
```

Comment:
revert here as well (edit: there's a few instances on the README where we want to keep TA when referring to the client.